content/docs/introducing-jsx.md

```diff
@@ -1,36 +1,36 @@
 ---
 id: introducing-jsx
-title: Introducing JSX
-permalink: docs/introducing-jsx.html
+title: Pagpapakilala sa JSX
+permalink: docs/pagpapakilala-sa-jsx.html
 prev: hello-world.html
 next: rendering-elements.html
 ---
 
-Consider this variable declaration:
+Mangyaring tingnan ang variable declaration na ito:
 
 ```js
 const element = <h1>Hello, world!</h1>;
 ```
 
-This funny tag syntax is neither a string nor HTML.
+Ang syntax na ito ay hindi isang string o HTML.
```

Comment:
`Para bang hindi matanto kung ito ba ay string o HTML.` - suggestion

Reply:
I put it that way to reflect the straightforward tone of the English documentation. I think we should keep it, unless the others have a say.